docs/configuration/cluster_manager/cds.rst

```diff
@@ -59,6 +59,8 @@ CDS has a statistics tree rooted at *cluster_manager.cds.* with the following st
   :header: Name, Type, Description
   :widths: 1, 1, 2
 
+  config_reload, Counter, Total API fetches that resulted in a config reload due to a different config
   update_attempt, Counter, Total API fetches attempted
   update_success, Counter, Total API fetches completed successfully
   update_failure, Counter, Total API fetches that failed (either network or schema errors)
+  version, Gauge, Hash of the contents from the last successful API fetch
```

Comment:
Don't we also want to report the actual `version_info` as provided by v2 xDS?

Reply:
I don't think we can provide that via a gauge, given that version_info is a string;

Reply:
we could attempt atoul on the first 6 chars similar to the server's version info and fallback to hashing. i'm not sure that it's worth it though.